includes/api/v1/class-wc-rest-webhooks-controller.php

```diff
@@ -133,6 +126,94 @@ public function register_routes() {
 		) );
 	}
 
+	/**
+	 * Check whether a given request has permission to read taxes.
```

Comment:
All the docblocks in this file are about taxes and they should be about webhooks. :P